### PR TITLE
Feature/image uploads

### PR DESCRIPTION
**Pull request by codestun**
WIP because S3 is pending - image uploads for restaurants - need to add image to cuisines and ingredients as well and add S3 to hold image uploads for production instead of cloudinary.

### CODE REVIEW

File: app/models/restaurant.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Restaurant < ApplicationRecord
+  mount_uploader :image, ImageUploader
   belongs_to :cuisine
   has_many :reviews
   has_many :ingredient_reviews, through: :reviews
   has_many :ingredients, through: :ingredient_reviews
   geocoded_by :address
   after_validation :geocode, if: :will_save_change_to_address?
-  has_one_attached :photo
```
Comment (georgkloeck) [inline app/models/restaurant.rb:8]: why do we not need this line anymore?

File: Gemfile [hunk 2]
```diff
@@ -5,61 +5,43 @@ ruby '2.6.6'
...
 
 group :test do
-  # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
   gem 'selenium-webdriver'
-  # Easy installation and use of web drivers to run system tests with browsers
   gem 'webdrivers'
 end
 
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
```
Comment (georgkloeck) [inline Gemfile:64]: I would prefer to keep the comments, can you pls change the gemfile so the comments stay?

File: app/models/restaurant.rb (hunk 1 above)
Comment (georgkloeck) [reply app/models/restaurant.rb:8]: cloudinary still uses active storage

Comment (codestun) [reply app/models/restaurant.rb:8]: Are you still using cloudinary?

**Pull request merged**